examples/AeroShield/AeroShield_PID/AeroShield_PID.ino

```diff
@@ -0,0 +1,105 @@
+/*
+  AeroShield closed-loop PID response example
+  PID feedback control for the aeropendulum.
+  
+  This example initializes the sampling and PID control 
+  subsystems from the AutomationShield library. You may
+  select whether the reference is given by the potentiometer
+  or you want to test a predetermined reference trajectory. 
+  Upload the code to your board, then open the Serial
+  Plotter function in your Arduino IDE. 
+  
+  Tested with Arduino Uno,
+  
+   This code is part of the AutomationShield hardware and software
+  ecosystem. Visit http://www.automationshield.com for more
+  details. This code is licensed under a Creative Commons
+  Attribution-NonCommercial 4.0 International License.
+
+  Created by  
+  1ast update: 13.03.2022.
+*/
+
+#include "AeroShield.h"               // Include main library  
+#include <Sampling.h>                 // Include sampling library
+
+#define MANUAL 0                      // Choose manual reference using potentiometer (1)  automatic reference trajectory (0) 
+#define KP 1.7                        // PID Kp constant
+#define TI 3.8                        // PID Ti constant
+#define TD 0.25                       // PID Td constant
+
+float startangle=0;           //  Variable for storing 0° angle in raw format
+float lastangle=0;            //  Variable needed for mapping of the pendulum's angle
+float pendulumAngle;          //  Variable used for storing the actual angle of the pendulum in degrees
+
+unsigned long Ts = 3;                 // Sampling period in milliseconds
+unsigned long k = 0;                  // Sample index
+bool nextStep = false;                // Flag for step function
+bool realTimeViolation = false;       // Flag for real-time sampling violation
+
+int i=i;                      // Section counter
+int T=1000;                    // Section length in ms
+float R[]={45.0,23.0,75.0,32.0,58.0,10.0,35.0,19.0,9.0,43.0,23.0,65.0,15.0,80.0};   // Reference trajectory
+float r=0.0;                  // Reference (Wanted pendulum angle)
+float y = 0.0;                // Output (Current pendulum angle)
+float u = 0.0;                // Input (motor power)
+
+void setup() {                                                  //  Setup - runs only once
+  Serial.begin(250000);                                         //  Begin serial communication
+  AeroShield.begin(AeroShield.detectMagnet());                  //  Initialise AeroShield board

```

Comment:
Same as in the open loop example. Im thinking this should be just begin() and the rest is done automatically